src/index.js

```diff
@@ -15,6 +15,8 @@ require('./components/roomcolor.js');
 require('./components/ui-raycaster.js');
 require('./components/gifcapture.js');
 
+require('aframe-mirror-component');
```

Comment:
the component is not included, is missing the file or the package.json I believe